src/page-modules/assistant/layout.tsx

```diff
@@ -54,6 +59,13 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     getTransportModeFilter,
   );
 
+  useEffect(() => {
+    tripQuery.transportModeFilter =
```

Comment:
Maybe `setValuesWithLoading()` should be used here as well? Like we looked at some time ago with some other changes. Or maybe not? I guess if the `setValuesWithLoading` is used we get an (possibly) unwanted route change.

And it's important to check whether `transportModeFilter` is set already, then the `isUncheckedByDefault` should not be taken into account and the filter should remain as is, since there's a chance the user has set the filters themselves. Haven't tested this code, but that might be how this works with the optional chaining and nullish coalescing.